tests.sh

```diff
@@ -217,24 +241,43 @@ dump_attach_info(){
 create_pool(){
     pmsg "Creating pool"
 
-    cmd="dmg -o $DAOS_CONTROL_YAML pool create --scm-size $POOL_SIZE"
-    echo $cmd
-    echo
-    DAOS_POOL="$(timeout --signal SIGKILL ${POOL_CREATE_TIMEOUT} ${cmd})"
-    check_cmd_timeout $? "dmg pool create"
+    HOST=$(head -n 1 ${CLIENT_HOSTLIST_FILE})
+    echo HOST ${HOST}
+    dmg_cmd="dmg -o ${DAOS_CONTROL_YAML} pool create --scm-size ${POOL_SIZE}"
+    cmd="clush -w ${HOST} --command_timeout ${POOL_CREATE_TIMEOUT} -S
+        \"${dmg_cmd}\""
+
+    pmsg "CMD: ${dmg_cmd}"
+    DAOS_POOL="$(eval ${cmd})"
+    RC=$?
```

Comment:
Why do you delay error handling ? Do you expect something interesting to be printed with DAOS_POOL, POOL_UUID and POOL_SVC upon "dmg pool create" failure ??

Reply:
Not really, I just copy-pasted the error handling statement at the end of the function as per the boilerplate on pretty much all the functions. If you want, I can move the if statement at L261 to L254 after printing DAOS_POOL which is the output of the command.